Iceberg.package/MCBitbucketRepository.extension/instance/getOrCreateIcebergRepository.st

```diff
@@ -0,0 +1,15 @@
+*Iceberg-Adapters
+getOrCreateIcebergRepository
```

Comment:
shouldn't this be moved to its parent?

Reply:
I was asking that myself but wasn't sure

Reply:
so?

Reply:
I can move it up

Reply:
done